routes/routes.js

```diff
@@ -0,0 +1,209 @@
+const School = require("../School");
+const express = require("express");
+const router = express.Router();
+const bodyParser = require("body-parser");
```

Comment:
You wouldn't need to require body-parser here, you would only require it on `app.js`